packages/backend/src/utils/parsers/quadlet-dryrun-parser.ts

```diff
@@ -39,6 +39,16 @@ export class QuadletDryRunParser extends Parser<RunResult & { exitCode?: number
     return Object.values(this.services);
   }
 
+  // https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Service%20Templates
+  protected isTemplate(path: string): boolean {
+    const filename = basename(path);
+
+    const separator = filename.lastIndexOf('.');
+    if (separator === -1) throw new Error('service name do not have .service extension');
```

Comment:
maybe include the service name in the error to see which one must have the ".service" extension ?

+ typo => "does not have"

Reply:
And this error is thrown where there is no "." (dot), it does not check for "service" extension.